VideoLocker/res/layout/drawer_navigation.xml

```diff
@@ -1,125 +1,129 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/grey_nav_background">
+    xmlns:tools="http://schemas.android.com/tools">
 
-    <FrameLayout
-        android:id="@+id/name_layout"
+    <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_alignParentTop="true"
-        android:foreground="@drawable/light_selectable_box_overlay">
+        android:layout_height="match_parent"
+        android:background="@color/grey_nav_background">
 
-        <LinearLayout
+        <FrameLayout
+            android:id="@+id/profile_layout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:background="@color/grey_name_background"
-            android:gravity="center"
-            android:orientation="horizontal"
-            android:padding="@dimen/edx_margin">
-
-            <de.hdodenhof.circleimageview.CircleImageView
-                android:id="@+id/profile_image"
-                style="@style/profile_image"
-                android:layout_width="@dimen/drawer_profile_image_diameter"
-                android:layout_height="@dimen/drawer_profile_image_diameter"
-                android:layout_marginEnd="@dimen/edx_margin"
-                android:layout_marginRight="@dimen/edx_margin"
-                app:civ_border_width="@dimen/drawer_profile_image_border_width"
-                tools:src="@drawable/profile_photo_placeholder" />
+            android:layout_alignParentTop="true"
+            android:foreground="@drawable/light_selectable_box_overlay">
 
             <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:orientation="vertical">
-
-                <TextView
-                    android:id="@+id/name_tv"
-                    style="@style/regular_white_text"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:textSize="@dimen/drawer_username_text"
-                    tools:targetApi="17"
-                    tools:text="username" />
-
-                <TextView
-                    android:id="@+id/email_tv"
-                    style="@style/regular_white_text"
-                    android:layout_width="wrap_content"
+                android:background="@color/grey_name_background"
+                android:gravity="center"
+                android:orientation="horizontal"
+                android:padding="@dimen/edx_margin">
+
+                <de.hdodenhof.circleimageview.CircleImageView
+                    android:id="@+id/profile_image"
+                    android:contentDescription="@string/label_my_profile"
```

Comment:
> It doesn't read back the user and email

Hmm. On the website, there is an explicit link to "Profile" from the nav menu. Perhaps adding an explicit entry to the list would be better? That would also make it more discoverable for sighted users, since it's not always intuitive that you can tap the user/email to get to your profile (we could still support that for the heck of it, but we would assume accessible users will use the explicit link below, and not worry about making it clear that this is a link)